README.md

```diff
@@ -471,7 +471,7 @@ Then for each `SourceSet` plugin adds following tasks:
 - `ktlint[source set name]SourceSetCheck` - generates reports and prints issues into Gradle console based on lint check found errors.
   This task execution depends on `loadKtlintReporters` and `runKtlintCheckOver[source set name]SourceSet` tasks execution outputs
 - `runKtlintFormatOver[source set name]SourceSet` - tries to format according to the code style every Kotlin file in given `SourceSet`
-- `ktlint[source set name]SourceSetCheck` - generates reports and prints issues into Gradle console based on found non-formattable errors.
+- `ktlint[source set name]SourceSetFormat` - generates reports and prints issues into Gradle console based on found non-formattable errors.
```

Comment:
good catch! I think the description should be updated as well to indicate that the source files will be formatted.

Reply:
Resolved I think